docs-kits/kits/DCM-Kit/page_adoption-view.md

```diff
@@ -38,11 +38,11 @@ Implementing the DCM standards will add the following core capabilities to busin
 
 ## Standards
 
-The DCM standard comprises three main elements:
+The DCM standard consists of three main elements:
 
-- **Core Business logic** – to focus on a **common understanding** of the **DCM processes** among the participants. **Version 1.0.0** (Release 3.1) [CX-0046 DCM Process & Core Business Logic](https://catena-x.net/de/standard-library)
-- **Data model** – to align the structure of the shared data and enable the adoption of the core business logic without misinterpretation. **Version 1.0.0** (Release 3.1) [CX-0047 DCM Data Model Material Demand & Capacity Group](https://catena-x.net/de/standard-library)
-- **APIs** – to make sure that data are transmitted and consumed properly as well as are identically interpreted across companies. **Version 1.0.0** (Release 3.1) [CX-0048 DCM API Material Demand & Capacity Group](https://catena-x.net/de/standard-library)
+- **Core Business logic** – to focus on a **common understanding** of the **DCM processes** among the participants. **Version 1.0.0** (Release 24.03) [CX-0128 Demand and Capacity Management Data Exchange](https://catena-x.net/de/standard-library)
+- **Data model** – to align the structure of the shared data and enable the adoption of the core business logic without misinterpretation. **Version 1.0.0** (Release 24.03) [CX-0128 Demand and Capacity Management Data Exchange](https://catena-x.net/de/standard-library)
+- **APIs** – to make sure that data are transmitted and consumed properly as well as are identically interpreted across companies. **Version 1.0.0** (Release 24.03) [CX-0128 Demand and Capacity Management Data Exchange](https://catena-x.net/de/standard-library)
 
 ### POWER of STANDARD
```

Comment:
Are those statements really necessary as H3? Because they are showing up in the right navigation and we want to keep this as lean as possible.

Reply:
Changed to ** **bold** ** -> Linter error message can be ignored